site/content/docs/developing/domain.en.md

```diff
@@ -23,7 +23,7 @@ http:
 ```
 
 !!!info
-    Currently, you can only use aliases under the domain you specified when creating the application. We'll make this feature more powerful in the future by allowing you to import certificates and use any aliases!
+    If you want to use aliases out of the domain you specified when creating the application. Please [import the certificates](../commands/env-init.en.md#what-are-the-flags) when creating the environment!
```

Comment:
nit: can we update it so that this is a tip and with the following text:
```
!!!tip
   If the application wasn't initialized with a domain, you can [import existing certificates](...) to your environment instead.
```